app/controllers/UserController.scala

```diff
@@ -149,15 +147,15 @@ class UserController @Inject()(val messagesApi: MessagesApi)
   // REST API
   def list = SecuredAction.async { implicit request =>
     UsingFilters(
-      Filter("includeAnonymous", (value: Boolean, el: User) => value || !el.isAnonymous, default = Some("false")),
```

Comment:
Why is the `includeAnonymous` Filter removed?

Reply:
the backend does not support anonymous users anymore (was for amazon mturk users).